decidim-meetings/app/permissions/decidim/meetings/permissions.rb

```diff
@@ -84,13 +84,18 @@ def can_decline_invitation?
       end
 
       def can_create_meetings?
-        component_settings&.creation_enabled_for_participants? && public_space_or_member?
+        (component_settings&.creation_enabled_for_participants? && can_participate?) || initiative_authorship?
       end
 
-      def public_space_or_member?
+      def can_participate?
+        context[:current_component].participatory_space.can_participate?(user)
+      end
+
+      def initiative_authorship?
         participatory_space = context[:current_component].participatory_space
 
-        participatory_space.private_space? ? space_member?(participatory_space, user) : true
+        participatory_space.is_a?(Decidim::Initiative) &&
```

Comment:
The same as before, to avoid errors check that the initiatives module is available or ask directly to the participatory space